resources/views/start.blade.php

```diff
@@ -0,0 +1,106 @@
+@extends('layouts.default')
+
+@section('content')
+<h1>Start Your Adventure!</h1>
+<div class="well">
+<h2>Want to test your skills in a shuffled Hyrule? You've come to the right place!</h2>
+
+<!--<div class="panel panel-info">
```

Comment:
blade syntax for commenting, will assure that the copy isn't even sent to browser
```
{{-- 
...
--}}
```